packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -82,6 +82,189 @@ describe('using cli', () => {
+        yarn add db@link:./generated/client
+        \`\`\`
+
+        More information: https://pris.ly/d/custom-output
+
+        2. Start using Prisma Client
+        \`\`\`
+        import { PrismaClient } from 'db'
+        const prisma = new PrismaClient()
+        \`\`\`
+
+        More information: https://pris.ly/d/client
+
+        ┌─────────────────────────────────────────────────────────────┐
+        │  Deploying your app to serverless or edge functions?        │
+        │  Try Prisma Accelerate for connection pooling and caching.  │
+        │  https://pris.ly/cli/accelerate                             │
+        └─────────────────────────────────────────────────────────────┘
+
+      `)
+    }
+  })
+
+  it('should display the right npm command for custom outputs', async () => {
+    ctx.fixture('custom-output-npm')
+    const data = await ctx.cli('generate')
+
+    if (typeof data.signal === 'number' && data.signal !== 0) {
+      throw new Error(data.stderr + data.stdout)
+    }
+
+    if (getClientEngineType() === ClientEngineType.Library) {
+      expect(data.stdout).toMatchInlineSnapshot(`
+        Prisma schema loaded from prisma/schema.prisma
+
+        ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        npm i db@./generated/client
+        \`\`\`
+
+        More information: https://pris.ly/d/custom-output
+
+        2. Start using Prisma Client
+        \`\`\`
+        import { PrismaClient } from 'db'
+        const prisma = new PrismaClient()
+        \`\`\`
+
+        More information: https://pris.ly/d/client
+
+        ┌─────────────────────────────────────────────────────────────┐
+        │  Deploying your app to serverless or edge functions?        │
+        │  Try Prisma Accelerate for connection pooling and caching.  │
+        │  https://pris.ly/cli/accelerate                             │
+        └─────────────────────────────────────────────────────────────┘
+
+      `)
+    }
+  })
+
+  it('should display the right pnpm command for custom outputs', async () => {
+    ctx.fixture('custom-output-pnpm')
+    const data = await ctx.cli('generate')
+
+    if (typeof data.signal === 'number' && data.signal !== 0) {
+      throw new Error(data.stderr + data.stdout)
+    }
+
+    if (getClientEngineType() === ClientEngineType.Library) {
+      expect(data.stdout).toMatchInlineSnapshot(`
+        Prisma schema loaded from prisma/schema.prisma
+
+        ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        pnpm add db@link:./generated/client
+        \`\`\`
+
+        More information: https://pris.ly/d/custom-output
```

Comment:
`https://pris.ly/d/custom-output` does not exist? I get a redirect to the landing page (`https://www.prisma.io/`)

Reply:
Yes, this is work that needs to be done by our DAs